HoughLines/adapter.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2022, Kirill Ivanov, Anastasiia Kornilova and Dmitrii Iarosh
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cv2
+import numpy as np
+
+from pathlib import Path
+from typing import Any, List
+
+from common.adapter.common_adapter import CommonAdapter
+from common.image_metadata import ImageMetadata
+from common.dataset.line_dataset import LineDataset
+from common.prediction import Prediction
+
+
+class Adapter(CommonAdapter):
+    def __init__(
+        self,
+        image_path: Path,
+        output_path: Path,
+        lines_output_directory: Path,
+        scores_output_directory: Path,
+    ):
+        super().__init__(
+            image_path,
+            output_path,
+            lines_output_directory,
+            scores_output_directory,
+        )
+        self.canny_first_threshold = 50
```

Comment:
If this is default parameters, let's add comment for this. The same is for other algorithms.